components/playlist/browser/resources/reducers/player_reducer.ts

```diff
@@ -136,9 +147,18 @@ const playerReducer: Reducer<PlayerState | undefined> = (
           throw new Error("Couldn't find the index of the current item ")
         }
 
+        let currentItem = null
```

Comment:
Optional:
```suggestion
        const currentItem = items.at(currentIndex - 1)
```

Reply:
awesome, I completely forgot about the negative index.